client.js

```diff
@@ -36,13 +37,18 @@ function DHT (opts) {
   this._verify = opts.verify || null
   this._host = opts.host || null
   this._interval = setInterval(rotateSecrets, ROTATE_INTERVAL)
+  this._bucketCheckInterval = null
   this._hash = opts.hash || sha1
 
   this.listening = false
   this.destroyed = false
   this.nodeId = this._rpc.id
   this.nodes = this._rpc.nodes
 
+  this.nodes.on('ping', function (nodes) {
+    self._checkNodes(nodes)
```

Comment:
it just happens when the bucket is full and wants to avoid a split: https://github.com/tristanls/k-bucket/blob/master/index.js#L126